app/components/banner/banner.css

```diff
@@ -6,6 +6,14 @@
   align-items: start;
 }
 
+.banner .banner-content > p:first-child {
```

Comment:
these margin overrides (basically content styling) feel like they'd be better placed in settings.css since the input child element for the banner is just an arbitrary div